src/client/cli/cmd/launch.cpp

```diff
@@ -107,9 +107,10 @@ mp::ParseCode cmd::Launch::parse_args(mp::ArgParser* parser)
     QCommandLineOption diskOption(
         {"d", "disk"},
         QString::fromStdString(fmt::format("Disk space to allocate. Positive integers, in "
-                                           "bytes, or with K, M, G suffix.\nMinimum: {}, default: {}.",
+                                           "bytes, or with K, M, G suffix.\nMinimum: {}, default: the bigger of {} "
+                                           " and the image size.",
```

Comment:
```suggestion
                                           "bytes, or with K, M, G suffix.\nMinimum: {}, default: {}, or the default "
                                           "image size, if larger.",
```

Not entirely happy with this wording, either…

Reply:
Is another option to leave it as it was?